configs/sites/gaea-c6/modules.yaml

```diff
@@ -0,0 +1,7 @@
+modules:
+  default:
+    enable::
+    - lmod
+    lmod:
+      include:
+      - python
```

Comment:
Not in this PR, but just as a note. We should at some point remove "Python" from the default exclude lists, and then remove all the manual include settings (basically each site has this now). This is a legacy setting from the early days of spack-stack when we didn't build Python but used an external Python instead.